utils/testing/mount.js

```diff
@@ -15,7 +15,9 @@ const unmountAll = () => {
     for (const key of wrappersToUnmount) {
         const wrapper = ACTIVE_WRAPPERS[key];
         delete ACTIVE_WRAPPERS[key];
-        wrapper.unmount();
+        if (wrapper.first().length) {
+            wrapper.unmount();
+        }
```

Comment:
After upgrading `enzyme` calling `.unmount()` failed b/c a couple of our tests do this manually (b/c we're actually testing the what happens when unmounting occurs).  When a component is unmounted `wrapper.first().length` will be `0`.